interface_library/AzureBot_Test.py

```diff
@@ -16,57 +17,117 @@ def send_value(self,value):
+                if rightJoy_LR[0] < 0
+                #joystick has been moved to the left
+                value = self.send_value(leftJoy_UD)
+                packet_rightThruster = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(255)
+                self.ser.write(packet_rightThruster)
+                packet_leftThruster = chr(1) + chr(7) + chr((self.value).encode("latin")) + chr(255)
+                self.ser.write(packet_leftThruster)
+
+
+            #previous code (reviewed and accepted - IGNORE)
+            '''
             while True:
-                
-                #coding the Left thruster (sendSystemsLes)
-                
-                if leftJoy_LR < [43,43]:
-                    value = send_value(leftJoy_LR)
-                    packet_leftJoy_stopped = chr(1) + chr(7) + chr((self.value).encode("latin")) + chr(12)
+                #coding the thrusters forward and back (sendSystemsLes)
+                if leftJoy_UD < [43,43]:
+                    value = self.send_value(leftJoy_UD)
+                    packet_leftJoy_stopped = chr(1) + chr(7) + chr((self.value).encode("latin")) + chr(255)
                     self.ser.write(packet_leftJoy_stopped)
                 else:
-                    value = send_value(leftJoy_LR)
-                    packet_leftJoy = chr(1) + chr(7) + chr((self.value).encode("latin")) + chr(12) 
+                    value = self.send_value(leftJoy_LR)
+                    packet_leftJoy = chr(1) + chr(7) + chr((self.value).encode("latin")) + chr(255) 
                     self.ser.write(packet_leftJoy)
 
                 #coding the Right thruster
                 if rightJoy_LR < [43,43]:
-                    value = send_value(rightJoy_LR)
-                    packet_rightJoy_stopped = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(12)
+                    value = self.send_value(rightJoy_LR)
+                    packet_rightJoy_stopped = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(255)
                     self.ser.write(packet_rightJoy_stopped)
                 else:
-                    packet_rightJoy = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(12) 
+                    packet_rightJoy = chr(1) + chr(6) + chr((self.value).encode("latin")) + chr(255) 
                     self.ser.write(packet_rightJoy)
+                '''
 
-                #servo claw code - finalized with packet value
+                #servo claw code - finalized with packet value. chr(11) tells systems to switch off the servo and chr(12) is an empty byte
                 if servoRotate == True:
-                    packet_servoRotate = chr(1) + chr(10) + chr(12)
+                    value = self.send_value(servoRotate)
+                    packet_servoRotate = chr(1) + chr(8) + chr(12) + chr(255)
                     self.ser.write(packet_servoRotate)
+                else:
+                    packet_servoRotate_off = chr(1) + chr(8) + chr(11) + chr(255)
```

Comment:
This variable is never mentioned again, what is this for? How can it be useful? Hint; look at the other similar if-statements